core/src/main/java/resource/context/admin/debug/Debug.cfc

```diff
@@ -51,7 +51,17 @@
 		<cfargument name="debugging" required="true" type="struct">
     	<cfthrow message="implement function output():string">
     </cffunction>
-    
-    
-    
+
+	<cfscript>
+		variables.serverTimingHeaders = [];
+
+		function addServerTimingHeader(metric, name, timeMs){
+			arrayAppend(variables.serverTimingHeaders, '"#arguments.name#" - #arguments.metric#=#arguments.timeMs#');
```

Comment:
have you tried this out in chrome to see if it works? 

the format for the server timing headers has changed since my original commit
https://github.com/zspitzer/lucee-loganalyzer/blob/refactoring/source/cfml/plugins/renderUtils.cfc#L54

Reply:
you can see these in the timing panel when you open a request under network in chrome dev tools

Reply:
@zspitzer, I'll check it & Let You Know

Reply:
@cfmitrah is that still ready to go for 5.3? if so can you please redirect it to 6.0. i do not wanna make that kind of change to 5.3